third_party/flower/src/proto/flwr/proto/node.proto

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Adap GmbH. All Rights Reserved.
```

Comment:
Is this a duplicate of the flower repo?

Reply:
Yes. These proto definitions are from the flower repo